vnext/src/Libraries/StyleSheet/PlatformColorValueTypes.windows.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict-local
+ */
+
+'use strict';
+
+import type {ColorValue} from './StyleSheetTypes';
+import type {ProcessedColorValue} from './processColor';
+
+export opaque type NativeColorValue = {
+  windowsbrush?: string,
+};
+
+export const PlatformColor = (...names: Array<string>): ColorValue => {
+  // We dont support fallback colors right now, so no point in sending more than the first color across the bridge
+  return {windowsbrush: names[0]};
+};
```

Comment:
Isn't this neat how WIndows can define a custom behavior for this function that works throughout the stack? It seems like this is something that would benefit from some documentation in core for other platforms to know how to do it. Who would be the right person to ask to help with that?